src/resource_prover.rs

```diff
@@ -100,17 +100,17 @@ impl ResourceProver {
     /// Start generating a resource proof in a background thread
     pub fn handle_request(
         &mut self,
-        pub_id: PublicId,
+        pub_info: PublicInfo,
         seed: Vec<u8>,
         target_size: usize,
         difficulty: u8,
-        log_ident: String,
+        log_infoent: String,
```

Comment:
I think this should be left as `log_ident` or changed to `log_info`.  Same in a few places below.